orangecontrib/text/tests/test_concordances.py

```diff
@@ -0,0 +1,199 @@
+import unittest
```

Comment:
Just a heads up, once #198 gets merged this file should be moved to `orangecontrib.text.widgets.tests`.